### ISSUE TITLE
Change emoji section icon color?

### ISSUE THREAD

**Issue by robzet** [ealush/emoji-picker-react#399]
Hello, is it possible to change the emoji section icon color from light blue? 

<img width="399" alt="Screenshot 2024-02-23 at 07 01 18" src="https://github.com/ealush/emoji-picker-react/assets/2285718/566edb43-1345-4ae3-8b76-e9c5af49905d">

**Comment by robzet** [comment]
And this is not on the roadmap to address? Im thinking it should be common to want to use your own color scheme to make the component feel integrated.

**Comment by dkrefta** [comment]
I was trying to change too the anchors colors ( active and hover), but following now this issue I know that will be probably impossible just with CSS haha
![image](https://github.com/ealush/emoji-picker-react/assets/29361657/52996048-4066-4b73-9322-7c4f97ee6d2c)

**Comment by ayushgupta-1** [comment]
Any update on this ? seems that it has not been resolved yet ?

**Comment by ealush** [comment]
Yes — that light blue is the `--epr-category-icon-active-color` CSS variable (default `#6aa8de`). You can restyle it without custom icons:

```css
.epr-main {
  --epr-category-icon-active-color: your-accent-color;
}
```

See CSS_VARIABLES.md (Category Navigation section) for the full list. If you want entirely different icons, pass the `categoryIcons` prop (or per-category `icon` in the `categories` config) — documented in CUSTOMIZATION.md and PROPS.md.

Closing as answered; feel free to reopen with specifics if neither covers your case.
